test/spec/exporters/jhipster_application_exporter.spec.js

```diff
@@ -175,13 +199,7 @@ describe('JHipsterApplicationExporter', () => {
               jhipsterVersion: '4.9.0'
             })
           );
-          content = JSON.parse(fs.readFileSync(path.join('.yo-rc.json'), { encoding: 'utf8' }));
-        });
-        after(() => {
-          fs.unlinkSync(path.join('.yo-rc.json'));
-        });
-
-        it('adds the read content to the exported application', () => {
+          const content = JSON.parse(fs.readFileSync(path.join('.yo-rc.json'), { encoding: 'utf8' }));
```

Comment:
I don't like having logic inside an it, "it"s should only have "expect"s, and setup code inside a before/beforeEach